test/integration/test_itde_manager_in_docker_container.py

```diff
@@ -250,11 +250,12 @@ def run_test():
         secrets = Secrets(db_file=Path("secrets.sqlcipher"), master_password="test")
         env_info, cleanup_func = api.spawn_test_environment(environment_name="TestDemoDb")
         try:
-            bring_itde_up(secrets, env_info)
-            with open_pyexasol_connection(secrets) as conn:
+            with patch('exasol_integration_test_docker_environment.lib.api.spawn_test_environment'):
```

Comment:
What is the reason for that? Do you want to use it for checking if it gets called?

Reply:
If `bring_itde_up` tries to create a new instance of the DockerDB the test will fail. To succeed the instance we work with must be real, not a mock.

Reply:
Ah, you mean because it would create a mock, we would enter garbage into the secrets and wouldn't be able to connect to the DB. Maybe add a comment for explanation and we are good to go